app/src/org/commcare/activities/CommCareVerificationActivity.java

```diff
@@ -276,6 +282,9 @@ public CustomProgressDialog generateProgressDialog(int taskId) {
                             Localization.get("verification.checking"),
                             taskId);
             dialog.addProgressBar();
+            if (fromSettings || fromManager) {

```

Comment:
@phillipm Hey, the PR looks fine code-wise, but I'm not sure if it addresses the situation Matt was talking about. It sounds from the ticket like he's frustrated by the fact that you can't close out of the MM validation screen that pops up *automatically* immediately after install. But it looks from this line like the Cancel button still wouldn't be available in that case. Let me know if you think I'm misunderstanding Matt's description or the code, but otherwise I would probably vote that it's actually fine to just have the Cancel button there in all instances of this activity.